scripts/scrape_elections.py

```diff
@@ -0,0 +1,56 @@
+from io import BytesIO
+import xlrd
+from pprint import pprint
+import pandas as pd
+from aiohttp import ClientSession
+from json import load, dump
+from asyncio import run
+
+
+
+def book_pandas(book: BytesIO):
+    workbook: xlrd.Book = xlrd.open_workbook(file_contents=book, ignore_workbook_corruption=True)
+    sheet = workbook.sheet_by_index(0)
+    rows = sheet.get_rows()
+    total = []
+    subtables = {}
+    for i in range(3):
+        next(rows)
+    subtables['Total'] = total
+    cur_row = next(rows)
+    while cur_row:
+
+        ward = cur_row[0].value
+        print(ward)
+        cur_row = next(rows)
+        sub_table = []
+        try:
+            while not all([cell.ctype in (xlrd.XL_CELL_EMPTY, xlrd.XL_CELL_BLANK) for cell in cur_row]):
+                sub_table.append([cell.value for cell in cur_row])
+                cur_row = next(rows)
+        except StopIteration:
+            pass
+        cols = sub_table[0]
+        print(sub_table)
+        print(cols)
+        for i in range(len(cols)):
```

Comment:
'cols = [col if col != '%' else cols[i-1] + " %" for i, cols in enumerate(cols)]'